app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("The Person has not been vaccinated.")
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("The vaccine of the "
+                                       "person has expired.")
+        elif ("wearing_a_mask" not in visitor
+              or visitor.get("wearing_a_mask") is False):
+            raise NotWearingMaskError("The person is not wearing a mask.")
```

Comment:
Code Style: Unnecessary `elif` usage. When you use `raise` in your `if` clause, you don't need to use `elif` for the next condition, because if the `if` condition was met, the function would exit at the `raise`. So, you can use `if` for each condition. This makes the code cleaner and easier to read.

Reply:
agree